src/community/contribution-criteria/index.md.njk

```diff
@@ -107,3 +107,19 @@ The working group reviews the implementation to make sure it is usable, consiste
     ]
   ]
 }) }}
+
+
+## Developing a community resource
+
+Before linking from the Design System to a [community resource](/community/resources) the Design System team will review it to make sure that:
```

Comment:
As mentioned, it’d be good if the lead-in line to the bullets is focused more on the user rather than us (something like: “We review all community resources before we link to them. Before you submit your resource, please make sure:” (But don’t worry about doing that now because it would make it inconsistent with the other lead-in lines on the page. Consistency for now is probably better.)